### PR TITLE
VTAdmin: Support for conclude txn and abandon age param

### PR DESCRIPTION
**Pull request by beingnoble03**
## Description
- Adds support for conclude transaction in VTAdmin.
- Adds an optional dropdown for selecting `abandonAge`, if it's empty, we use the default abandon age.
- The dropdown carries the following options: `5sec`, `30sec`, `1min`, `5min`, `15min` and `1hr`.

## Related Issue(s)
- #16766

## Screenshots
Screenshot 2024-09-25 at 1 04 32 AM

## Checklist

-   [ ] "Backport to:" labels have been added if this change should be back-ported to release branches
-   [ ] If this change is to be back-ported to previous releases, a justification is included in the PR description
-   [ ] Tests were added or are not required
-   [ ] Did the new or modified tests pass consistently locally and on CI?
-   [ ] Documentation was added or is not required

### CODE REVIEW

File: go/vt/vtadmin/http/request.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func (r Request) ParseQueryParamAsUint32(name string, defaultVal uint32) (uint32
 }
 
 // ParseQueryParamAsInt32 attempts to parse the query parameter of the given
-// name into a uint32 value. If the parameter is not set, the provided default
+// name into a int32 value. If the parameter is not set, the provided default
```
Comment (beingnoble03) [inline go/vt/vtadmin/http/request.go:81]: typo :)

**Review by rohit-nayak-ps — APPROVED** [review]

File: go/vt/vtadmin/http/transactions.go [hunk 2]
```diff
@@ -23,13 +23,32 @@ import (
 )
 
 // GetUnresolvedTransactions implements the http wrapper for the
-// /transactions/{cluster_id}/{keyspace} route.
+// /transactions/{cluster_id}/{keyspace}[?abandon_age=] route.
```
Comment (notfelineit) [inline go/vt/vtadmin/http/transactions.go:26]: 🥳

File: go/vt/vtadmin/api.go [hunk 3]
```diff
@@ -531,6 +532,27 @@ func (api *API) CompleteSchemaMigration(ctx context.Context, req *vtadminpb.Comp
...
+	c, err := api.getClusterForRequest(req.ClusterId)
+	if err != nil {
+		return nil, err
+	}
+
+	cluster.AnnotateSpan(c, span)
+
+	if !api.authz.IsAuthorized(ctx, c.ID, rbac.KeyspaceResource, rbac.GetAction) {
+		return nil, nil
+	}
```
[the remark this answers is not in the document]
Comment (beingnoble03) [reply go/vt/vtadmin/api.go]: Done.

**Review by harshit-gangal — COMMENTED** [review]
Sorry this fall off my radar.
We should keep the default set for Abandon Age to 15 mins here.

**Review by harshit-gangal — APPROVED** [review]

**Pull request merged**